docker-compose.yml

```diff
@@ -1,48 +1,58 @@
-db:
-  image: postgres:9.4
-  ports:
-    - "5432"
+version: '2'
 
-app:
-  build: .
-  command: echo "Hello, world!"
-  volumes:
-    - .:/app
-  stdin_open: true
-  environment:
-    BUNDLE_PATH: "/box"
+services:
+  
+  app:
+    build: .
+    command: echo "Hello, world!"
+    volumes:
+      - .:/app
+    stdin_open: true
+    environment:
+      BUNDLE_PATH: "/box"
+      SKYLIGHT_AUTHENTICATION: "test"
+      INSTAGRAM_CLIENT_ID: "test"
+      INSTAGRAM_CLIENT_SECRET: "test"
```

Comment:
These environment variables will be set by the ".env" file which is created as part of the bootstrap script. You might have to rebase master to get the change.